charts/prometheus-nats-exporter/templates/deployment.yaml

```diff
@@ -49,6 +49,9 @@ spec:
             {{- if .Values.config.metrics.gatewayz }}
             - "-gatewayz"
             {{- end }}
+            {{- if .Values.config.metrics.jsz }}
+            - "-jsz=all"
```

Comment:
Is it possible to define other values then "all" for jsz?

Reply:
It seems it is
https://github.com/nats-io/prometheus-nats-exporter/pull/141/files#diff-f4a6a94945eeecf45c42373c6a61aabe85cea9c9f563548d66469d6ed7d1f64dR238-R246